Inc/cli/cli.hpp

```diff
@@ -5,42 +5,56 @@
 #ifndef PHONE_CLI_HPP
 #define PHONE_CLI_HPP
 
-#include "modem/modem.hpp"
-#include "rotary_reader/rotary_dial.hpp"
-#include "cli/option.hpp"
-
 #include <QObject>
+#include <QMainWindow>
 #include <iostream>
-#include "screen.hpp"
 #include <memory>
 #include <ncurses.h>
 #include <unordered_map>
+#include <QSharedPointer>
+#include "screen.hpp"
+#include "cli/option.hpp"
+#include "rotary_reader/rotary_dial.hpp"
 
 using ScreenMap = QHash<QString, QSharedPointer<Screen>>;
 
+class ModemController;
```

Comment:
Is it necessary here?

Reply:
Seems that cli doesn't have modem controller

Reply:
Yep, this is a necessary forward declaration here, it helps avoid circular dependencies